frontend/src/components/Wheel/index.js

```diff
@@ -1,65 +1,82 @@
 import { useEffect, useState } from "react";
-import SpinButton from "./button";
 import styles from "./index.module.css";
+import "./index.css";
+
+import { WHEEL_CONFIG } from "constants/Subvisual.js";
 
 const Wheel = () => {
-  const [wheelInfo, setWheelInfo] = useState({});
+  const [prizesInfo, setPrizesInfo] = useState([]);
+  const [rotate, setRotate] = useState("circle");
+
+  async function startRotate() {
+    setRotate("circle start-rotate");
+
+    setTimeout(() => {
+      setRotate("circle start-rotate stop-rotate");
+    }, Math.floor(Math.random() * 10000) + 1);
+  }
+
+  function wheelStyle(i) {
+    let backgroundColor;
+
+    if (i % 2 === 0) {
+      backgroundColor = WHEEL_CONFIG.colors.main;
+    } else {
+      backgroundColor = WHEEL_CONFIG.colors.secondary;
+    }
+
+    const style = {
+      transform: `rotate(${(360 / prizesInfo?.length) * i}deg) skewY(-60deg)`,
+      background: `${backgroundColor}`,
+    };
+
+    return style;
+  }
 
   useEffect(() => {
-    fetch(`${process.env.REACT_APP_BACKEND_PATH}/wheel`)
+    fetch(`${process.env.REACT_APP_BACKEND_PATH}/prizes`)
       .then((response) => {
         if (response.ok) {
           return response.json();
         } else {
           throw response;
         }
       })
-      .then((data) => setWheelInfo(data));
+      .then((data) => {
+        setPrizesInfo(data);
+      });
   }, []);
 
   return (
-    <div>
-      <table>
-        <tr>
-          <th>ID</th>
-          <th>Name</th>
-          <th>Logo</th>
-          <th>Prizes</th>
-          <th>Color1</th>
-          <th>Color2</th>
-        </tr>
-
-        <tr key={wheelInfo?.id}>
-          <td>{wheelInfo?.id}</td>
-          <td>{wheelInfo?.name}</td>
-          <td>{wheelInfo?.logo}</td>
-
-          <td>
-            {wheelInfo?.prizes?.map((prize) => {
-              return (
-                <div key={prize.id}>
-                  <span>Name: {prize.name}</span>
-                  <span> Percentage: {prize.percentage}</span>
-                  <span> Image: {prize.image}</span>
-                </div>
-              );
-            })}
-          </td>
+    <>
+      <h1 className={styles.center}>Prizes</h1>
+      <div className={styles.arrow}></div>
 
-          <td>{wheelInfo?.config?.main_color}</td>
-          <td>{wheelInfo?.config?.secondary_color}</td>
-        </tr>
-      </table>
+      <ul className={rotate}>
+        {prizesInfo.map((prize, index) => {
+          return (
+            <li
+              className={styles.prize}
+              key={prize?.id}
+              style={wheelStyle(index)}
+            >
+              <div contentEditable="true" spellCheck="false">
+                <span className={styles.text}>{prize?.name}</span>
+                <img
+                  src={prize?.image}
+                  alt={prize?.name}
+                  className={styles.img_prizes}
```

Comment:
We're using CSS modules: `imgPrizes` Though I don't understand what image prizes are? If it's only one image?